schema/sqitch.plan

```diff
@@ -363,3 +363,4 @@ views/ciip_incentive_per_facility 2021-06-16T17:12:43Z Pierre Bastianelli <pierr
 @v2.9.0 2021-06-24T16:12:00Z Matthieu Foucault <matthieu@button.is> # release v2.9.0
 swrs_functions/refresh_swrs_version_data [swrs_functions/refresh_swrs_version_data@v2.9.0] 2021-07-05T16:49:44Z Dylan Leard <dylan@button.is> # Migration: key on combination of swrs_facility_id and reporting_year
 @v2.10.0 2021-07-07T23:51:18Z Dylan Leard <dylan@button.is> # release v2.10.0
+enum_operators 2021-07-08T21:17:17Z Aro, Naomi IIT:EX <naomiaro@NEM020064> # Enum operators for Metabase.
```

Comment:
- I would rename this to `database_functions/dd_enum_comparison_operators`, and move/rename the deploy/revert/verify files accordingly
- You can set your name and emai with
```
sqitch config --user user.name 'Marge N. O’Vera'
sqitch config --user user.email 'marge@example.com'
```
(see https://sqitch.org/docs/manual/sqitch-configuration/)